05.ios/05.solving-problems/09.how-to-avoid-compatibility-issues-with-FaceTime/docs.en.md

```diff
@@ -0,0 +1,21 @@
+---
+title: 'How to avoid compatibility issues with FaceTime'
+taxonomy:
+    category:
+        - docs
+   
+---
+
+It turned out that Full-Tunnel mode can interfere not only with compatibility with other VPN applications, but also with FaceTime.
```

Comment:
It turned out that Full-Tunnel mode can interfere not only  --> Full-Tunnel mode may interfere not only